src/JLSOFile.jl

```diff
@@ -79,13 +107,22 @@ function Base.:(==)(a::JLSOFile, b::JLSOFile)
         a.version == b.version &&
         a.julia == b.julia &&
         a.image == b.image &&
-        a.pkgs == b.pkgs &&
+        a.manifest == b.manifest &&
         a.format == b.format &&
         a.compression == b.compression &&
         a.objects == b.objects
     )
 end
 
 Base.names(jlso::JLSOFile) = collect(keys(jlso.objects))
-
-# TODO: Include a more detail summary method for displaying version information.
+function Base.getproperty(jlso::JLSOFile, attr::Symbol)
+    if attr === :pkgs
+        @warn "pkgs property is deprecated, use .manifest instead"
```

Comment:
I almost want to keep this, but that is another story

Reply:
You mean keep the `pkgs` field? I think this would be better as a function that's easier to deprecate moving forward.